themes/bootstrap3/js/common.js

```diff
@@ -240,19 +240,31 @@ var VuFind = (function VuFind() {
   };
 
   var loadHtml = function loadHtml(_element, url, data, success) {
-    var $elem = $(_element);
-    if ($elem.length === 0) {
+    var element = document.querySelector(_element);
+  
+    if (!element) {
       return;
     }
-    $.get(url, typeof data !== 'undefined' ? data : {}, function onComplete(responseText, textStatus, jqXhr) {
-      if ('success' === textStatus || 'notmodified' === textStatus) {
-        $elem.html(updateCspNonce(responseText));
-      }
-      if (typeof success !== 'undefined') {
-        success(responseText, textStatus, jqXhr);
+  
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
+  
```

Comment:
The `onload` handler should be defined before `open()` is called. Otherwise there's at least in theory a possibility that the request completed before the onload handler is set up.